config/manager/kustomization.yaml

```diff
@@ -3,3 +3,10 @@ resources:
 
```

Comment:
Not required to be committed.

Reply:
@nbalacha how about adding unwanted files to gitignore?

Reply:
I don't think it is required. If it makes things less confusing, we can merge the temporary files as they will be overwritten anyway by the make targets.

Reply:
Yes as you mentioned it's less confusing and helps both contributors and maintainers to keep track of it.